app/controllers/businessactivities/RiskAssessmentController.scala

```diff
@@ -59,21 +59,19 @@ class RiskAssessmentController @Inject() (val dataCacheConnector: DataCacheConne
 
             for {
               businessActivities <- dataCacheConnector.fetch[BusinessActivities](BusinessActivities.key)
-              _ <- dataCacheConnector.save[BusinessActivities](BusinessActivities.key, businessActivities.riskAssessmentPolicy(data))
-            } yield redirectDependingOnEdit(edit, ControllerHelper.isAccountancyServicesSelected(Some(businessMatching)))
-
+              _ <- dataCacheConnector.save[BusinessActivities](BusinessActivities.key, businessActivities.riskAssessmentHasPolicy(data))
+            } yield redirectDependingOnAccountancyServices(ControllerHelper.isAccountancyServicesSelected(Some(businessMatching)), data)
           }
         } recoverWith {
           case _: IndexOutOfBoundsException => Future.successful(NotFound(notFoundView))
         }
       }
   }
 
-  private def redirectDependingOnEdit(edit: Boolean, accountancyServices: Boolean) = edit match {
-    case true => Redirect(routes.SummaryController.get())
-    case false => accountancyServices match {
+  private def redirectDependingOnAccountancyServices(accountancyServices: Boolean, data: RiskAssessmentHasPolicy) =
+    accountancyServices match {
```

Comment:
Match on both?

Reply:
Swav said to just use data as an if case as its only used once. Guess its down to preference?